test/test-qunit.js

```diff
@@ -19,7 +19,7 @@ test('basic test', function() {
 
     // Check that noConflict restored the original log
     ok(typeof log === "function", "log is a function");
-    ok(log === QUnit.log, "log is Qunit.log");
+    ok(log === window.customLogFunction, "log is customLogFunction");
```

Comment:
The original reason this test was added was as a specific repro for an issue with QUnit's custom log function. That made sense to deal with the issue properly, but it's been fixed upstream in QUnit for a while now (which is why your upgrade will have highlighted this), and I think now moving this to a more generic version is indeed sensible.

If we're doing that though, this test is now totally unrelated to QUnit, so we could actually just turn this into a Jasmine test that does the same thing, within the rest of the suite, and drop the QUnit dep entirely.